Foundation/NSHTTPCookieStorage.swift

```diff
@@ -65,7 +65,7 @@ open class HTTPCookieStorage: NSObject {
         guard !FileManager.default.fileExists(atPath: path) else { return true }
 
         do {
-            try FileManager.default.createDirectory(atPath: path, withIntermediateDirectories: false, attributes: nil)
+            try FileManager.default.createDirectory(atPath: path, withIntermediateDirectories: true, attributes: nil)
```

Comment:
Currently, if there was a non-existent parent directory in the 'path', this would return false and the cookie storage would happen in the current working directory. The expected behaviour is to create the parent directories as well.